Build error on Linux with GCC 6.1

Compiling Git at commit https://github.com/red-eclipse/base/commit/99bf21684c51a0d8d3ed88f932dbe0608f9da114 on Arch Linux with GCC 6.1.1 returns this error:

```
g++ -O3 -fomit-frame-pointer -ffast-math -Wall -fsigned-char -fno-exceptions -fno-rtti -I. -Ishared -Iengine -Igame -Ienet/include -Isupport -I/usr/X11R6/include `sdl2-config --cflags`   -c -o engine/server.o engine/server.cpp
engine/server.cpp: In function ‘void setlocations(bool)’:
engine/engine.h:28:33: error: unable to find string literal operator ‘operator""a’ with ‘const char [12]’, ‘long unsigned int’ arguments
 #define sup_var(a) VERSION_VNAME"_"a
                                 ^
engine/server.cpp:1587:30: note: in expansion of macro ‘sup_var’
     const char *dir = getenv(sup_var("DATADIR"));
                              ^~~~~~~
engine/server.cpp: In function ‘void setverinfo(const char*)’:
engine/engine.h:28:33: error: unable to find string literal operator ‘operator""a’ with ‘const char [12]’, ‘long unsigned int’ arguments
 #define sup_var(a) VERSION_VNAME"_"a
                                 ^
engine/server.cpp:1704:34: note: in expansion of macro ‘sup_var’
     const char *vbranch = getenv(sup_var("BRANCH"));
                                  ^~~~~~~
<builtin>: recipe for target 'engine/server.o' failed
make: *** [engine/server.o] Error 1
```

--- a/src/engine/engine.h
+++ b/src/engine/engine.h
@@ -25,7 +25,7 @@ extern char *systemuser, *systemhost;
 
 #define sup_platform(a) (a >= 0 && a < MAX_PLATFORMS)
 #define sup_arch(a) (a == 32 || a == 64)
-#define sup_var(a) VERSION_VNAME"_"a
+#define sup_var(a) VERSION_VNAME"_" a
 
 extern const char *platnames[MAX_PLATFORMS], *platlongnames[MAX_PLATFORMS];
 #define plat_name(a) (sup_platform(a) ? platnames[a] : "unk")

